### PR TITLE
Basic Travis CI precise build script

### PR DESCRIPTION
**Pull request by jayvdb**
Uses ppa:staticfloat/juliareleases for Julia.
Uses Travis apt addon for R and clang.

Closes https://github.com/coala/coala/issues/3191

### CODE REVIEW

**Comment by jayvdb** [comment]
Green build at https://travis-ci.org/jayvdb/coala/builds/182846769

**Comment by jayvdb** [comment]
Gah.  Another "Merge" commit https://github.com/coala/coala/commit/ad3bfc023086ac8e4

maybe we need to solve https://github.com/coala/coala-bears/issues/1037 first.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+language: python
+python:
+  - 3.4
+  - 3.5
```
Comment (sils) [inline .travis.yml]: can you add 3.6 while we're at it?

Comment (jayvdb) [inline .travis.yml]: [no](https://travis-ci.org/jayvdb/coala/builds/182831634).  well, `3.6` hasnt been added to Travis yet. maybe they have it available under a different label, but I havent looked.  we can add it soon, I am sure.

Comment (Mixih) [inline .travis.yml]: Is that going to slow the the testing process even further though?

Comment (jayvdb) [inline .travis.yml]: oi! The Travis builds are faster than Circle CI :P
But, no, Travis runs the jobs in parallel, so adding more jobs doesnt impact elapsed time until there are more than 5 concurrent jobs, and I have seen up to 15 concurrent OSS jobs in Travis off-peak times.

Comment (sils) [inline .travis.yml]: use `3.6-dev`

Comment (sils) [inline .travis.yml]: not sure, you may need to give it as a string: https://docs.travis-ci.com/user/languages/python/#Default-Python-Version

Comment (Mixih) [inline .travis.yml]: It's working now in the latest build (I think)

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3206/commits/527c7a46eae46594142ec67959ef93e70932e711

**Comment by Mixih** [comment]
ack 527c7a4

**Pull request merged**